INSTALL.md

```diff
@@ -457,8 +460,8 @@ Download and install following packages:
 | Tool |Website|
 |---------------|---------------|
 |CMake |https://cmake.org/files/v3.12/cmake-3.12.3-win64-x64.msi|
```

Comment:
Also, I think it'd be better to replace the table with a list:

 - [CMake](https://cmake.org/files/v3.12/cmake-3.12.3-win64-x64.msi)
 ....